ci-operator/config/openshift/prometheus-config-reloader/master.json

```diff
@@ -0,0 +1,36 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    }
+  },
+  "canonical_go_repository": "github.com/coreos/prometheus-operator",
+
+  "images": [
+    {
+      "from": "base",
```

Comment:
We generally create one definition per repo, so you should add this into the openshift/prometheus-operator config as a new image.